dist/formats/person/frontend/schema.json

```diff
@@ -448,6 +448,10 @@
         "analytics_identifier": {
           "$ref": "#/definitions/analytics_identifier"
         },
+        "body": {
+          "description": "Biography of the person",
+          "type": "string"
```

Comment:
Does this need to be `body_html_and_govspeak`?

https://github.com/alphagov/govuk-content-schemas/blob/master/formats/help_page.jsonnet#L9?

Reply:
Yes, that works :)